### PR TITLE
Amazon ML hyperparameter optimization example

### PR DESCRIPTION
**Pull request by alexandraj777**
Copying example from https://github.com/alexandraj777/machine-learning-samples/tree/master/hyperparameter-optimization to sigopt-examples

### CODE REVIEW

File: amazon-ml-hyperparameter-optimization/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Performing Hyperparameter Optimization with Amazon Machine Learning
+
+This sample code builds a hyperparameter optimization pipeline for Amazon Machine Learning using the latest AWS SDK for Python (Boto 3). The user can optionally specify hyperparameters upfront for manual tuning or use [SigOpt](https://www.sigopt.com)'s API for Bayesian optimization.
+
+This example is directly based off of Amazon's [K-Fold Cross Validation](https://github.com/awslabs/machine-learning-samples/tree/master/k-fold-cross-validation) example.
```
Comment (pfhayes) [inline amazon-ml-hyperparameter-optimization/README.md]: the LICENSE file from the original repo needs to be included

File: amazon-ml-hyperparameter-optimization/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+### Configure SigOpt Credentials (optional)
+
+If want to use SigOpt to optimize your hyperparameters faster and better than tuning by hand, sign up for a free trial on our [website]([SigOpt](https://www.sigopt.com)) and grab your API token from your [user profile](https://www.sigopt.com/user/profile).
+
+### Get the Code
+
+Get the samples by cloning this repository.
+
+```
+  git clone https://github.com/alexandraj777/machine-learning-samples.git
```
Comment (pfhayes) [inline amazon-ml-hyperparameter-optimization/README.md]: update this url

**Comment by sc932** [comment]
🌽 👍  very cool :)

**Comment by alexandraj777** [comment]
🌽 👍  I GET IT

**Pull request merged**